00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/hub.yaml proposal for themes.md

```diff
@@ -0,0 +1,303 @@
+---
+publish: true
+---
+
+# `hub.yaml` proposal for theme designers
```

Comment:
just a precaution, but sometimes you wrote `hub.yml`, sometimes `hub.yaml` – we should be consistent about that, or better, the scripter of thingies should look for both, just in case. Cause I am pretty sure some theme devs gonna write it one way, others the other way.

Reply:
Huh weird, search and replace didn't find any `yml` strings in the file